mscr-ui/src/modules/form/index.tsx

```diff
@@ -363,7 +376,7 @@ export default function FormModal({
             setSubmitAnimationVisible(false);
           }
         );
-      } else if (modalType == ModalType.RegisterNewMscr) {
+      } else if (modalType == ModalType.RegisterNewMscr) {//what is register new MSCR?
```

Comment:
If you look at the definition of the ModalType:
```
export enum ModalType {
  RegisterNewFull = 'REGISTER_NEW_FULL',
  RegisterNewMscr = 'REGISTER_NEW_MSCR',
  RevisionFull = 'REVISION_FULL',
  RevisionMscr = 'REVISION_MSCR',
  MscrCopy = 'MSCR_COPY',
}
```
RegisterNewMscr is registering something in MSCR-format, where there's no file sent, as opposed to RegisterNewFull, where there should also be a file attached. In practice it always means creating (and registering) a new crosswalk, since that's always MSCR-format.

Reply:
Thanks :) I was wondering whar this action :)